src/frontend/.stylelintrc.json

```diff
@@ -2,6 +2,7 @@
   "extends": "stylelint-config-recommended",
   "plugins": ["stylelint-scss"],
   "rules": {
+    "no-empty-source": null,
```

Comment:
It doesn't seem like we added any empty source files. What am I missing?

Reply:
We have several empty tags and I'm not sure what the best way to deal with them is:

```
[1] src/views/LandingPageV2/components/Footer/style.ts
[1]   45:42  ✖  Unexpected empty source   no-empty-source
[1]   51:45  ✖  Unexpected empty source   no-empty-source
[1]  145:33  ✖  Unexpected empty source   no-empty-source
```